EpiAware/test/predictive_checking/fast_approx_for_r.jl

```diff
@@ -0,0 +1,73 @@
+#=
+# Fast approximation for `r` from `R₀`
+I use the negative moment generating function (MGF).
```

Comment:
Why isn't this in the docs for the function?

Reply:
I've added some now.